examples/terraform-aws-rds-example/main.tf

```diff
@@ -0,0 +1,89 @@
+resource "aws_vpc" "example" {
```

Comment:
need multiple subnets in different AZs for a subnet group for RDS. since creating a subnet in an existing VPC will require some weird CIDR magic to define the second subnet if it doesn't exist, went in for the simpler approach of creating a new VPC.
from performance perspective, vpc and subnets are negligible as compared to the db_instance - approx 5-6 minutes each for creation and termination.

please resolve if you agree

Reply:
`aws_subnet_ids` returns multiple subnets in different AZs if you point it at the default VPC: https://github.com/gruntwork-io/terratest/blob/master/examples/terraform-redeploy-example/main.tf#L274-L286

Reply:
done. please review and resolve.